### PR TITLE
Added wget prometheus test command

### PR DESCRIPTION
**Pull request by jcharcalla**
wget command that can be ran from the prometheus container to verify network connectivity.

### CODE REVIEW

File: custom-prom.md [hunk 1]
```diff
@@ -109,6 +109,13 @@ kubectl exec kubecost-cost-analyzer-<UID> -c cost-analyzer-frontend -n kubecost
 
 If the config file is not returned, this is an indication that an incorrect Prometheus address has been provided. If a config file is returned from one pod in the cluster but not the Kubecost pod, then the Kubecost pod likely has its access restricted by a network policy, service mesh, etc.
 
+**Context Deadline Exceeded**: Network policies, Mesh networks, or other security related tooling can block network traffic between Prometheus and Kubecost which will result in the Kubecost scrape target state as being down in the Prometheus targets UI. To assist in troubleshooting this type of error you can use the `wget` command from within the Prometheus Server container to try and reach the Kubecost target manually. Note the "namespace" and "deployment" name in this command may need updated to match your environment.\
+
+When succesfull this command should return all of the Kubecost metrics. Failures may be indicative of the network traffic being blocked.
+```
+kubectl exec -it -n monitoring deployment/prometheus-server -c prometheus-server --  wget -S -O - http://kubecost-cost-analyzer.kubecost:9003/metrics
```
Comment (keithhand) [inline custom-prom.md:116]: I think this should be updated to use our default deployment name, `deployment/kubecost-prometheus-server`. And possibly our default example namespace of `kubecost`, for ease when copy/pasting. Ex.: `kubectl exec -it -n kubecost deployment/kubecost-prometheus-server -c prometheus-server --  wget -S -O - http://kubecost-cost-analyzer.kubecost:9003/metrics`

Comment (jcharcalla) [reply custom-prom.md]: Good thinking, I have made the change.

File: custom-prom.md [hunk 2]
```diff
@@ -109,6 +109,13 @@ kubectl exec kubecost-cost-analyzer-<UID> -c cost-analyzer-frontend -n kubecost
 
 If the config file is not returned, this is an indication that an incorrect Prometheus address has been provided. If a config file is returned from one pod in the cluster but not the Kubecost pod, then the Kubecost pod likely has its access restricted by a network policy, service mesh, etc.
 
+**Context Deadline Exceeded**: Network policies, Mesh networks, or other security related tooling can block network traffic between Prometheus and Kubecost which will result in the Kubecost scrape target state as being down in the Prometheus targets UI. To assist in troubleshooting this type of error you can use the `wget` command from within the Prometheus Server container to try and reach the Kubecost target manually. Note the "namespace" and "deployment" name in this command may need updated to match your environment, this example uses the default Kubecost Prometheus deployment.\
+
+When succesfull this command should return all of the Kubecost metrics. Failures may be indicative of the network traffic being blocked.
```
Comment (srpomeroy) [inline custom-prom.md:114]: spelling

```suggestion
When successful this command should return all of the Kubecost metrics. Failures may be indicative of the network traffic being blocked.
```

**Review by srpomeroy — CHANGES_REQUESTED** [review]

**Review by srpomeroy — APPROVED** [review]
LGTM

**Pull request merged**